http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -161,48 +159,44 @@ echo "LEONARDO_URL = ${LEONARDO_URL}"
-sudo runuser -l $VM_JUP_USER -c "wget -qP /anaconda/lib/python3.10/site-packages https://raw.githubusercontent.com/DataBiosphere/terra-docker/0ea6d2ebd7fcae7072e01e1c2f2d178390a276b0/terra-jupyter-base/custom/jupyter_delocalize.py"
-sudo runuser -l $VM_JUP_USER -c "sed -i 's/notebook.services/jupyter_server.services/g' /anaconda/lib/python3.10/site-packages/jupyter_delocalize.py"
-sudo runuser -l $VM_JUP_USER -c "sed -i 's/http:\/\/welder:8080/http:\/\/127.0.0.1:8081/g' /anaconda/lib/python3.10/site-packages/jupyter_delocalize.py"
-
-echo "------ Jupyter ------"
+##### JUPYTER SERVER  #####
+echo "------ Jupyter version: ${JUPYTER_DOCKER_IMAGE} ------"
 echo "Starting Jupyter with command..."
 
-echo "sudo runuser -l $VM_JUP_USER -c \"/anaconda/bin/jupyter server --ServerApp.base_url=$SERVER_APP_BASE_URL --ServerApp.websocket_url=$SERVER_APP_WEBSOCKET_URL --ServerApp.contents_manager_class=jupyter_delocalize.WelderContentsManager --autoreload &> /home/$VM_JUP_USER/jupyter.log\"" >/dev/null 2>&1&
+echo "docker run -d --restart always --network host --name jupyter \
+--entrypoint tail \
+--volume ${PD_DIRECTORY}:${NOTEBOOKS_DIR}/persistent_disk \
+--publish 8888:8888 \
+-e WORKSPACE_ID=$WORKSPACE_ID \
+-e WORKSPACE_NAME=$WORKSPACE_NAME \
+-e WORKSPACE_STORAGE_CONTAINER_URL=$WORKSPACE_STORAGE_CONTAINER_URL \
+-e STORAGE_CONTAINER_RESOURCE_ID=$WORKSPACE_STORAGE_CONTAINER_ID \
+$JUPYTER_DOCKER_IMAGE \
+-f /dev/null"
+
+#Run docker container with Jupyter Server
+#Override entrypoint with a placeholder (tail -f /dev/null) to keep the container running indefinitely.
+#The jupyter server itself will be started via docker exec after.
+#Mount the work directory to a new persistent_disk directory to maintain parity with the legacy DSVM experience
+#and avoid overwriting the docker NOTEBOOKS_DIR that contains python/conda
+docker run -d --restart always --network host --name jupyter \
+--entrypoint tail \
+--volume ${PD_DIRECTORY}:${NOTEBOOKS_DIR}/persistent_disk \
+--publish 8888:8888 \
+--env WORKSPACE_ID=$WORKSPACE_ID \
+--env WORKSPACE_NAME=$WORKSPACE_NAME \
+--env WORKSPACE_STORAGE_CONTAINER_URL=$WORKSPACE_STORAGE_CONTAINER_URL \
+--env STORAGE_CONTAINER_RESOURCE_ID=$WORKSPACE_STORAGE_CONTAINER_ID \
+$JUPYTER_DOCKER_IMAGE \
+-f /dev/null
 
-sudo runuser -l $VM_JUP_USER -c "/anaconda/bin/jupyter server --ServerApp.base_url=$SERVER_APP_BASE_URL --ServerApp.websocket_url=$SERVER_APP_WEBSOCKET_URL --ServerApp.contents_manager_class=jupyter_delocalize.WelderContentsManager --autoreload &> /home/$VM_JUP_USER/jupyter.log" >/dev/null 2>&1&
+echo 'Starting Jupyter Notebook...'
+echo "docker exec -d jupyter /bin/bash -c "/usr/jupytervenv/run-jupyter.sh ${SERVER_APP_BASE_URL} ${SERVER_APP_WEBSOCKET_URL} ${NOTEBOOKS_DIR}""
+docker exec -d jupyter /bin/bash -c "/usr/jupytervenv/run-jupyter.sh ${SERVER_APP_BASE_URL} ${SERVER_APP_WEBSOCKET_URL} ${NOTEBOOKS_DIR}"
 
-# Store Jupyter Server parameters for reboot processes
```

Comment:
Ohhh I need to add something similar back to fix the pause/resume issue that I am seeing :) 💡